app/scripts/filters/findProject.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .filter('findProject', function($filter) {
+    return function(items, searchString) {
+      // return early if no search string
+      if(!searchString) {
+        return items;
+      }
+
+      return _.filter(items, function(item) {
+        var searchStringLower = searchString.toLowerCase();
+
```

Comment:
@benjaminapetersen _.lowerCase() is only available in lodash v4.14.1. I cannot find an equivalent lower case function in v3.10.1.